python/tvm/target/target.py

```diff
@@ -313,10 +313,10 @@ def canon_target_map_and_host(target_map, target_host=None):
         Target.canon_target. The final target_map keys will capture the target_host in
         canonical form. Also returns the target_host in canonical form."""
         if target_host is not None:
-            warnings.warn(
-                "target_host parameter is going to be deprecated. "
-                "Please pass in tvm.target.Target(target, host=target_host) instead."
-            )
+            # warnings.warn(
```

Comment:
Is this intentional? I suppose it's not throwing warnings if used correctly, right?

Reply:
(definitely not an expert here)

Reply:
Rebased & fixed.